### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by twikista**
Added the login page for the user app

### CODE REVIEW

File: src/pages/Login/LoginForm/LoginForm.jsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+} from "./LoginForm.styles";
+import Checkbox from "../../../components/checkbox/Checkbox";
+import { EyeIconClose, EyeIconOpen } from "../EyeIcons/EyeIcons";
+import Text from "../../../components/Text/Text";
+import { useState } from "react";
+
+const LoginForm = ({ register, handleSubmit, handler, errors }) => {
+  const [showPassword, setShowPassword] = useState(false);
+  const passwordInputType = showPassword ? "text" : "password";
+  const emailRegex = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
```
[the remark this answers is not in the document]
Comment (JohnOmoseni) [reply src/pages/Login/LoginForm/LoginForm.jsx:24]: Please, Check this out

**Review by JohnOmoseni — COMMENTED** [review]
There is an issue with the regex that you're using to validate the email input field.

File: src/pages/Login/LoginForm/LoginForm.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (twikista) [reply src/pages/Login/LoginForm/LoginForm.jsx]: I have update the email regex

**Review by iamevaristus — COMMENTED** [review]
You did not make it responsive, also change the title to "Serch", theme color to "030001", the meta icon is not seen, the design template is not represented in the Serch which is in the background of the login form. Note, the S is capital letter and Se shows on the left side while rch shows on the right

**Comment by twikista** [comment]
fixed pending issues

**Review by JohnOmoseni — APPROVED** [review]

**Pull request merged**